### PR TITLE
Add act-as-exchange-transport-agent.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add act-as-exchange-transport-agent.yml

### CODE REVIEW

File: persistence/exchange/act-as-exchange-transport-agent.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    att&ck:
+      - Persistence::Server Software Component::Transport Agent [T1505.002]
+    references:
+      - https://learn.microsoft.com/en-us/exchange/mail-flow/transport-agents/transport-agents?view=exchserver-2019
+      - https://learn.microsoft.com/en-us/exchange/client-developer/transport-agents/how-to-create-a-deliveryagent-transport-agent-for-exchange-2013
+    examples:
+      - a301eadd2b665b696803e143dd4d657d71c56bbded2a3a1b96c5bcb83cc6796a:0x600000E
+  features:
+    - or:
+      - api: Microsoft.Exchange.Data.Transport.Smtp.SmtpReceiveAgent::add_OnConnectEvent
```
Comment (williballenthin) [inline persistence/exchange/act-as-exchange-transport-agent.yml:17]: if these are .NET features, perhaps add a feature `format: dotnet` to help make this clear?

**Review by williballenthin — APPROVED** [review]
lgtm

File: persistence/exchange/act-as-exchange-transport-agent.yml (hunk 1 above)
Comment (jtothej) [reply persistence/exchange/act-as-exchange-transport-agent.yml]: Thanks! Fixed in https://github.com/mandiant/capa-rules/pull/782/commits/7754ec8e07d34de0aa43ebd0244d79aa7bb5adfa

**Comment by mr-tz** [comment]
Thanks!

**Pull request merged**